packages/core/admin/admin/src/content-manager/history/components/VersionsList.tsx

```diff
@@ -143,6 +143,28 @@ const VersionCard = ({ version, isCurrent }: VersionCardProps) => {
   );
 };
 
+/* -------------------------------------------------------------------------------------------------
+ * PaginationButton
+ * -----------------------------------------------------------------------------------------------*/
+
+interface PaginationButtonProps {
+  page: number;
+  children: React.ReactNode;
+}
+
+const PaginationButton = ({ page, children }: PaginationButtonProps) => {
+  const [{ query }] = useQueryParams<{ id?: string }>();
+  const { id: _id, ...queryRest } = query;
```

Comment:
The linter doesn't complain since the prefix `_` creates an allowed unused variable. I didn't know that. Is that what you want? If so how come?

Reply:
sorry I missed this comment. Yes I want to have an object that's the whole thing except id, because id isn't needed by the server, and passing it created unnecessary refetches when it changed. So I'm only destructuring id because I want to have an object without it. I could also use `delete` instead but it doesn't feel functional programming-y

Reply:
Oh right of course you want a new object without id 🤦‍♂️ , thanks for the comment :)